DependencyInjection/DoctrineExtension.php

```diff
@@ -164,9 +167,7 @@ protected function loadDbalConnection($name, array $connection, ContainerBuilder
             $profilingLoggerId = $profilingAbstractId . '.' . $name;
             $container->setDefinition($profilingLoggerId, new ChildDefinition($profilingAbstractId));
             $profilingLogger = new Reference($profilingLoggerId);
-            $container->getDefinition('data_collector.doctrine')
-                ->addMethodCall('addLogger', [$name, $profilingLogger])
-                ->replaceArgument(1, $connection['profiling_collect_schema_errors']);
```

Comment:
Doing this conditionally caused the bug because the default value is `true`: https://github.com/doctrine/DoctrineBundle/blob/155a870ecbd035963e6b450e0f235e69c48cab30/DataCollector/DoctrineDataCollector.php#L34